pkg/textparse/parse.go

```diff
@@ -131,6 +138,15 @@ func (p *Parser) Metric(l *labels.Labels) string {
 	return s
 }
 
+var escaper = byte('\\')
```

Comment:
`escaper` sounds like a type that has an `escape()` method. How about just inlining the `'\\'`?